proto/buf/alpha/registry/v1alpha1/repository_branch.proto

```diff
@@ -37,6 +37,14 @@ service RepositoryBranchService {
   rpc ListRepositoryBranches(ListRepositoryBranchesRequest) returns (ListRepositoryBranchesResponse) {
     option idempotency_level = NO_SIDE_EFFECTS;
   }
+  // GetDefaultBranch returns the branch name that is mapped to the main/BSR_HEAD.
+  rpc GetDefaultBranch(GetDefaultBranchRequest) returns (GetDefaultBranchResponse) {
```

Comment:
This RPC name conflicts with L28-29 in which we say it's not "default" but "main" :/

I like "default" better though.

Reply:
Ah that's true :< Hmm... yeah, I want to use the term default... tbh, I wish we could change the other RPC structure, lol... but it might be late for that. I think `Default` is a lot more explicit and clear for this use case though.

Reply:
Had an offline discussion, we also decided to go with `Current`, since this makes it clear it's the current/latest default branch, and does not reflect the history of default branches if it's ever been changed.

Reply:
Should the `is_main_branch` be removed? I think it may no longer be necessary with the addition of this new rpc. It also feels weird to have a field dedicated to indicate true only for one branch that is default(main).

Reply:
> And when listing the non-default, it could be filtered out by the client using the is_main_branch from L29.
ah sorry I missed this part Julian wrote. I guess it would be necessary if we decide to go in that direction. Let's chat in the meeting.